docs/modules/ROOT/pages/cops_style.adoc

```diff
@@ -9391,6 +9393,24 @@ assert_match(/foo/, bar)
 array.delete(e)
 ----
 
+[#includedmacropatterns_-___assert__-__refute__-stylemethodcallwithargsparentheses]
+==== IncludedMacroPatterns: ["^assert", "^refute"]
+
+[source,ruby]
+----
+# bad
+class TestClass
+  assert_equal 'test', x
+  refute_nil value
+end
+
+# good
+class TestClass
+  assert_equal('test', x)
+  refute_nil(value)
+end
+----
+
```

Comment:
Changes to the adoc files are automatically generated on release and should not be made within a PR.